### DISCUSSION TITLE
What is this startup setting/state/filter on apps?

### DISCUSSION THREAD

**Discussion by Mili8B** [Klocman/Bulk-Crap-Uninstaller#617] [Q&A] [answered]
![Screenshot 2024-05-30 183655](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/90237912/fc354a3c-e1ff-4d89-8ad1-87a1dc863870)

**Comment by Klocman** [comment, marked as answer]
"Yes" in this column means that some file of this application is launched on system startup or by task scheduler. See tools > startup manager.
